static/constants.json

```diff
@@ -27,6 +27,16 @@
   },
   "Primer::Alpha::ActionList::Heading": {
     "DEFAULT_SCHEME": "subtle",
+    "HEADING_LEVELS": [
```

Comment:
```suggestion
    "HEADING_LEVELS": Array.from({length: HEADING_MAX}, (_, i) => i + 1)
```

no big deal just a thought

Reply:
Sorry to disagree but I feel like this makes it harder to read 😅

Reply:
Hmm, so this constants.json file is auto-generated from constants found in the component source code. We'd have to make changes there in order for them to be reflected here.

However, I don't understand why we'd want to increment all the heading levels by 1. Wouldn't that give us `h2` - `h7`?

Reply:
`<h7>` is not a thing! 😅

Reply:
That code would output 1-6 it (its using index so it starts at 0). That being said if its autogenerated then its not needed.